pkg/util/ssh.go

```diff
@@ -174,6 +174,13 @@ func runSSHCommand(dialer sshDialer, cmd, user, host string, signer ssh.Signer)
 		Auth: []ssh.AuthMethod{ssh.PublicKeys(signer)},
 	}
 	client, err := dialer.Dial("tcp", host, config)
+	// Retry the dial a few times if it failed.
+	var retries = 3
```

Comment:
Using `wait.Poll` introduces a dependency cycle. Is there still a way to do it, or should I switch back to manual retries?